Sources/Paywall/Misc/Extensions/Date+IsoString.swift

```diff
@@ -9,31 +9,14 @@ import Foundation
 
 extension Date {
   var isoString: String {
-    if #available(iOS 11.0, *) {
-      let formatter = ISO8601DateFormatter()
-      formatter.formatOptions = [.withInternetDateTime, .withFractionalSeconds]
-      return formatter.string(from: self)
-    }
-
-    let formatter = DateFormatter()
-    formatter.calendar = Calendar(identifier: .iso8601)
-    formatter.locale = Locale(identifier: "en_US_POSIX")
-    formatter.timeZone = TimeZone(secondsFromGMT: 0)
-    formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSXXXXX"
+    let formatter = ISO8601DateFormatter()
+    formatter.formatOptions = [.withInternetDateTime, .withFractionalSeconds]
```

Comment:
Not sure if this is the case or not but are we sure this will produce the same format as the previous code?

Reply:
It will because all I did was remove the `#available(iOS 11.0, *)` tag. Since our minimum version is iOS 11.2 this won't be a problem. Looks like maybe you targeted iOS 10 in the past? We can also add tests to this.

Reply:
Ok just wanted to make sure, adding tests would be great!